jobs/nats/spec

```diff
@@ -40,6 +47,14 @@ properties:
     default: 30
   nats.tls.ca:
     description: CA cert used by NATS server to verify clients certificates (For Mutual TLS Connections)
+  nats.tls.health_monitor.certificate:
+    description: |
+      Certificate for establishing mutual TLS with NATS server. The Common-Name for the certificate
+      should be "default.hm.bosh-internal"
+  nats.tls.director.certificate:
+    description: |
+      Certificate for NATs mutual TLS client (Director client). The Common-Name for this certificate
+      should be "default.director.bosh-internal"
```

Comment:
> It would be good if we could provide/consume these via links. That would prevent operators from having to specify the values twice in the manifest or use global properties.
> 
> Would also reduce errors by ensuring the values match.

Never mind, terrible idea. It looks like the bosh cli does NOT support links at all, so this would not work when doing a `bosh create-env` if we changed it to links.